internal/filegreeter/greeter.go

```diff
@@ -0,0 +1,63 @@
+package filegreeter
+
+import (
+	"bufio"
+	"context"
+	"github.com/Tommy647/go_example/internal/greeter"
+	"io"
+	"log"
+	"os"
+)
```

Comment:
import ordering (stdlib, thirdpary, internal) 

Run `goimports` on file save to help with this, Goland has a FileWatchers tool in the settings

Reply:
Thanks, fixed